src/mlpack/tests/ne_test.cpp

```diff
@@ -0,0 +1,691 @@
+/**
+ * @file ne_test.cpp
```

Comment:
Can you split this test into smaller parts. Like ``link_gene_test.cpp, neuron_gene_test.cpp, genome_test.cpp, cne_test.cpp, neat_test.cpp``.